examples/hello-world-native/hello-indexer-native/src/main.rs

```diff
@@ -29,6 +29,7 @@ use fuel_indexer_utils::prelude::*;
 mod hello_world_native {
 
     async fn index_logged_greeting(event: Greeting, block: BlockData) {
+        log_info!("Handling {:?}", event);
```

Comment:
Would it be possible to change these to their `tracing` equivalent? So in this case instead of `log_info!`, just `info!` ?